src/migrations/20230908155325-fix-fei-root-causes.js

```diff
@@ -0,0 +1,163 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      await queryInterface.sequelize.query(`
+        -- Update workforce root causes for fei.
+        with ids as (
+            SELECT
+            argfr.id
+            FROM "Grants" gr
+            JOIN "Goals" goal
+                ON gr.id = goal."grantId"
+            JOIN "ActivityReportGoals" arg
+                ON goal."id" = arg."goalId"
+            JOIN "ActivityReportGoalFieldResponses" argfr
+                ON arg.id = argfr."activityReportGoalId"
+            WHERE "number" IN ('06CH011554', '06CH011558', '06CH010965', '06CH010720', '06HP000120', '06HP000296', '06CH011143'))
+            UPDATE "ActivityReportGoalFieldResponses"
+                SET "response" = '{Workforce}'
+            WHERE "id" IN (SELECT "id" FROM "ids");
+
+            with ids as (
+                SELECT
+                gfr.id
+                FROM "Grants" gr
+                JOIN "Goals" goal
+                    ON gr.id = goal."grantId"
+                JOIN "GoalFieldResponses" gfr
+                    ON goal.id = gfr."goalId"
+                WHERE "number" IN ('06CH011554', '06CH011558', '06CH010965', '06CH010720', '06HP000120', '06HP000296', '06CH011143'))
+                UPDATE "GoalFieldResponses"
+                    SET "response" = '{Workforce}'
+                WHERE "id" IN (SELECT "id" FROM "ids");
+
+            -- Update workforce, transportation root causes for fei.
+            with ids as (
+                SELECT
+                argfr.id
+                FROM "Grants" gr
+                JOIN "Goals" goal
+                    ON gr.id = goal."grantId"
+                JOIN "ActivityReportGoals" arg
+                    ON goal."id" = arg."goalId"
+                JOIN "ActivityReportGoalFieldResponses" argfr
+                    ON arg.id = argfr."activityReportGoalId"
+                WHERE "number" IN ('06CH011469', '06CH011503', '06CH011919'))
+                UPDATE "ActivityReportGoalFieldResponses"
```

Comment:
When I queried the prod snapshot I have from a few weeks ago, none of these had "ActivityReportGoalFieldResponses" set at all. 

Maybe we should ask HHS if it makes sense to add the data to the AR's that have already been approved or just the goals?

Reply:
Confirmed at stand up update both.